dedicated-examples/extended-error-tracking/README.MD

```diff
@@ -0,0 +1,55 @@
+# Extended Client-Side Error Tracking
+
+When a JavaScript error occurs on a tested web page, TestCafe stops test execution and posts an error message. Third-party code execution (e.g. controls, logging and analytics services) might cause unexpected errors that are not relevant to the tested app. You might want to filter those errors out to ignore them during the test.
+
+This example demonstrates how to extend TestCafe's standard error tracking and filter client-side errors that happen during a test.
+
+**Test Code**: [index.js](index.js)
+
+The test page has 2 buttons (`Error 1` and `Error 2`), each throws an error with `.level` and `.url` custom properties when clicked. These properties are later used to filter the errors and have the following values:
```

Comment:
I think here we don't have to say 'later' - the properties are obviously used after they are defined